modular_skyrat/modules/blueshield/code/modules/clothing/suits/armor.dm

```diff
@@ -1,10 +1,10 @@
 //blueshield armor
 /obj/item/clothing/suit/armor/vest/blueshield
-	name = "Blueshield's armored suit jacket"
-	desc = "An expensive suit jacket with reinforced fabric and an easily tucked in armored vest underneath."
+	name = "Blueshield's jacket"
+	desc = "An expensive kevlar-lined jacket with a golden badge, despite how heavy it looks it's reasonably light. Has ''NT'' embazloned on the back. There's a tight-fitting vest tucked in underneath."
```

Comment:
```suggestion
	desc = "An expensive kevlar-lined jacket with a golden badge, despite how heavy it looks it's reasonably light. Has \"NT\" embazloned on the back. There's a tight-fitting vest tucked in underneath."
```
This would look cleaner. Never underestimate the escape character.

Reply:
This is good.